pom.xml

```diff
@@ -664,7 +664,7 @@
         <carbon.extension.identity.authenticator.version>3.0.0</carbon.extension.identity.authenticator.version>
         <org.wso2.carbon.identity.association.account>5.1.5</org.wso2.carbon.identity.association.account>
         <identity.extension.utils>1.0.8</identity.extension.utils>
-        <carbon.identity.framework.version>5.25.124</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.161-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Any reason to go with snapshot?

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/3891/commits/828c51c15382db600153eb193bf7dc37e9feccc1